config/settings.yml

```diff
@@ -1417,6 +1417,21 @@ vanotify:
         claim_submission_failure_text: <%= ENV['vanotify__services__check_in__template_id__claim_submission_failure_text'] %>
         claim_submission_success_text: <%= ENV['vanotify__services__check_in__template_id__claim_submission_success_text'] %>
         claim_submission_timeout_text: <%= ENV['vanotify__services__check_in__template_id__claim_submission_timeout_text'] %>
+    dependents:
+      api_key: <%= ENV['vanotify__services__dependents__api_key'] %>
+      email:
+        received674:
+          template_id: <%= ENV['vanotify__services__dependents__email__received674__template_id'] %>
+        received686:
+          template_id: <%= ENV['vanotify__services__dependents__received686__template_id'] %>
+        received686c674:
+          template_id: <%= ENV['vanotify__services__dependents__received686c674__template_id'] %>
+        submitted674:
+            template_id: <%= ENV['vanotify__services__dependents__submitted674__template_id'] %>
+        submitted686:
+          template_id: <%= ENV['vanotify__services__dependents__submitted686__template_id'] %>
+        submitted686c674:
+          template_id: <%= ENV['vanotify__services__dependents__submitted686c674__template_id'] %>
```

Comment:
😵 I think you're missing the `__email__` portion of these paths.

Reply:
once you fix that, this PR should be good to go.

Reply:
Eek! Thanks for the catch!! It's fixed now